tests/ember_debug/object-inspector-test.js

```diff
@@ -757,14 +771,23 @@ module('Ember Debug - Object Inspector', function (hooks) {
       3,
       'Correctly merges properties'
     );
-    assert.equal(message.details[3].properties[1].name, 'hasChildren');
+
+    const hasChildren = A(message.details[3].properties).findBy(
+      'name',
+      'hasChildren'
+    );
+    const expensiveProperty = A(message.details[3].properties).findBy(
+      'name',
+      'expensiveProperty'
+    );
```

Comment:
```suggestion
    const hasChildren = message.details[3].properties.find((p) => p.name === 'hasChildren');
    const expensiveProperty = message.details[3].properties.find((p) => p.name === 'expensiveProperty');
```

Reply:
@rwjblue is there a reason not to use `Ember.A`?

Reply:
Ya, we don't need it 😃. Ember.A is about providing observeability, we aren't trying to observe and react to changes here. We just want to find a specific item. That's exactly what `.find` is for, and it results in the codebase being easier to follow (IMHO).

It is also quite likely that Ember.A itself will go away over time (in favor of specific mutation methods that themselves trigger the reactivity that we need), and in general we should prefer native methods for things whenever / where ever they exist.